cf_remote/utils.py

```diff
@@ -76,7 +76,7 @@ def above_package_path():
 
 
 def is_package_url(string):
-    return bool(re.match("https?://.+/.+\.(rpm|deb|msi)", string))
+    return bool(re.match("https?://.+/.+\.(rpm|deb|msi|tar\.gz|tgz)", string))
```

Comment:
Using raw strings `r""` you can omit the escape characters, which makes the regex a little easier to interpret.

Reply:
@larsewi good rule in general, but that's not what's happening here. The `\.` is not an escape that python will do anything with, so it will end up in the regex, you don't have to `\\.`. We actually want the `\.` in the regex, meaning specifically dot, not the normal meaning of a dot (wildcard).

In this case, adding the r for raw string makes no difference, there is nothing in that string which python will change;

```
$ python3
Python 3.8.5 (default, Jul 28 2020, 12:59:40)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> "https?://.+/.+\.(rpm|deb|msi|tar\.gz|tgz)"
'https?://.+/.+\\.(rpm|deb|msi|tar\\.gz|tgz)'
>>> r"https?://.+/.+\.(rpm|deb|msi|tar\.gz|tgz)"
'https?://.+/.+\\.(rpm|deb|msi|tar\\.gz|tgz)'
>>> "https?://.+/.+\.(rpm|deb|msi|tar\.gz|tgz)" == r"https?://.+/.+\.(rpm|deb|msi|tar\.gz|tgz)"
True
```

Reply:
https://github.com/cfengine/cf-remote/pull/8